src/ec/packages/ec-component-language-list/language-list-item.html.twig

```diff
@@ -19,10 +19,10 @@
 
 {# Internal logic - Process properties #}
 
-{% set _item_icon = 
+{% set _item_icon =
```

Comment:
Could you please double-check also the spacing between brackets?
- https://github.com/ec-europa/ecl-twig/pull/68/files#diff-4f104de808bce416807a253fbf41b36dR35
- https://github.com/ec-europa/ecl-twig/pull/68/files#diff-4f104de808bce416807a253fbf41b36dL36
- https://github.com/ec-europa/ecl-twig/pull/68/files#diff-4f104de808bce416807a253fbf41b36dR38